java/maven/src/org/netbeans/modules/maven/modelcache/MavenProjectCache.java

```diff
@@ -364,11 +411,16 @@ public static MavenProject getFallbackProject(File projectFile) throws Assertion
         newproject.setName(Bundle.LBL_Incomplete_Project_Name());
         newproject.setDescription(Bundle.LBL_Incomplete_Project_Desc());
         newproject.setFile(projectFile);
+        newproject.setContextValue(CONTEXT_FALLBACK_PROJECT, true);
         return newproject;
     }
     
     public static boolean isFallbackproject(MavenProject prj) {
-        return "error".equals(prj.getGroupId()) && "error".equals(prj.getArtifactId()) && Bundle.LBL_Incomplete_Project_Name().equals(prj.getName());
+        if ("error".equals(prj.getGroupId()) && "error".equals(prj.getArtifactId()) && Bundle.LBL_Incomplete_Project_Name().equals(prj.getName())) {
```

Comment:
whouldn't this method fit better into MavenProject class?

Reply:
`org.apache.maven.project.MavenProject` is not part of NetBeans project. I need some utility place to put this helper in; maybe the cache is not the best place -- will solve in subsequent PR with 'more appropriate fix' described at the top.

Reply:
sorry ... haven't noticed that - you're right